Kajabi/sage-lib
# [SAGE-457] Sage docs cleanup

## Description
Covers a laundry list of documentation site validation/accessibility/performance items to hold us over until the docs redesign.

### Home
- Fixes broken "skip to content" link
- Removes legacy screenshot [SAGE-721](https://kajabi.atlassian.net/browse/SAGE-721). Verified change with design
- Refactors and replaces hero background image
- Adds hero background image blur effect
- Removes devicons dependency, replacing with inline SVGs

### Search
- Adds missing label on form input

### General
- Fixes meta tag links rendering `%` character
- Corrects missing `alt` attribute in Sage logo (assistant/nav)
- "Invalid" iframe widths of `100%` replaced 
- Displays "skip to content" link when focused
- Removes "prettify" code styling on the majority of pages
- Removes `type="text/javascript"` attribute on scripts


## Screenshots

|  Before  |  After  |
|--------|--------|
|<img src="https://user-images.githubusercontent.com/816579/182697863-8f14d644-2577-4e75-90a8-bc8fc9470fb3.png" alt="" width="330">|<img src="https://user-images.githubusercontent.com/816579/183138180-af2b70d1-f180-42e2-a166-09cfb782d609.png" alt="" width="330">|


## Testing in `sage-lib`
1. View the [docs site homepage](http://localhost:4000/pages/index)
2. Confirm that the screenshot has been removed
3. Verify that the "How It Works" icons appear as expected
4. Scan the page using [WAVE](https://wave.webaim.org/extension/) or [axe](https://www.deque.com/axe/), and verify that any remaining issues are color-contrast related.


## Testing in `kajabi-products`

1. (**N/A**) Documentation site changes only


## Related
